gamemodes/terrortown/gamemode/shared/sh_lang.lua

```diff
@@ -25,21 +25,25 @@ local net = net
 local table = table
 local pairs = pairs
 
--- Add all lua files in our /lang/ dir
-local dir = GM.FolderName or "terrortown"
-local files = file.Find(dir .. "/gamemode/shared/lang/*.lua", "LUA")
+function LANG.SetupFiles(lang_path)
+	-- Add all lua files in our /lang/ dir
+	local files = file.Find(lang_path .. "*.lua", "LUA")
 
-for _, fname in pairs(files) do
-	local path = dir .. "/gamemode/shared/lang/" .. fname
+	for _, fname in pairs(files) do
+		local path = lang_path .. fname
 
-	-- filter out directories and temp files (like .lua~)
-	if string.Right(fname, 3) == "lua" then
-		util.IncludeClientFile(path)
+		-- filter out directories and temp files (like .lua~)
+		if string.Right(fname, 3) == "lua" then
+			util.IncludeClientFile(path)
 
-		MsgN("Included TTT language file: " .. fname)
+			MsgN("Included TTT language file: " .. fname)
+		end
 	end
 end
 
+-- load default TTT2 language files
```

Comment:
You are including in sv_ and cl_main too, why here again?

Reply:
I have only put this in a function. The TTT language system is designed this way. Not by me.

Reply:
But you are doing this 3 times, one time shared, one time for client, one time for server. That‘s two times for esch realm

Reply:
no. Look at the paths. It's once done on client and server for the TTT2 language files and once on client an server for addon language files